apps/codeforafrica/src/config/index.js

```diff
@@ -0,0 +1,26 @@
+const config = {
```

Comment:
This should be in `lib/index.js` and return as part of page static props. This way, we'll only need to change one place to return values from CMS when we get to that stage.